pkg/controllers/migration/virtualmachine.go

```diff
@@ -647,3 +655,24 @@ func (h *virtualMachineHandler) tidyUpObjects(vm *migration.VirtualMachineImport
 	}
 	return nil
 }
+
+// generateAnnotations will generate the harvester reference annotations: github.com/harvester/harvester/pkg/ref
+func generateAnnotations(vm *migration.VirtualMachineImport, vmi *harvesterv1beta1.VirtualMachineImage) (map[string]string, error) {
```

Comment:
can we try to reuse https://github.com/harvester/harvester/blob/master/pkg/controller/master/virtualmachine/pvc_util.go#L54-L62 if appliable?

Reply:
let's leave it as a future enhancement and consider making `harvester/harvester` func more flexible to adapt, thanks.